src/App.jsx

```diff
@@ -27,10 +27,10 @@ function App() {
   }, []);
   return (
     <>
-      <div className="fixed top-4 left-4 z-50">
+      <Header />
+      <div className="fixed top-4 right-4 z-50">
         <ThemeToggle />
       </div>
```

Comment:
We should move the theme toggle into the header component now so the logo can be aligned with it and doesn't clip it on mobile.